randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -145,7 +145,12 @@ Extra - asset_id: 3
           Any of the following:
               # Via bottom path
               Can Lay Bombs
-              Gravity Suit and Screw Attack
+              All of the following:
+                  Morph Ball
+                  Any of the following:
+                      Enabled Moat Tweaks
+                      # Destroy with Screw
+                      Gravity Suit and Screw Attack
 
```

Comment:
I looked at this in my upcoming PR. You can also just jump over, I labeled it as Movement (Intermediate). Like you're going to CWJ, but you can just land on the center platform and from there the path is trivial.

The Shinespark in the TODO is also possible. It requires 43 Energy, Disabled Entrance Randomizer, and a 1 tap short charge.

![image](https://github.com/randovania/randovania/assets/46821362/9a5ec1ca-9fe8-44c4-9cd4-d5e814403a63)

Reply:
2 taps*

Reply:
Interesting that it requires taps, would've thought that'd be possible without any

Since this PR is only focused on putting the patch resources in, and not additional logic, feel free to do that in your PR.